frontend/src/screens/Subject/EditionCard.tsx

```diff
@@ -5,16 +5,32 @@ import { colors } from '../../utils/colors';
 import { Button } from 'semantic-ui-react';
 import { subtitleFontSize } from '../../utils/sizes';
 import { useState } from 'react';
+import { deleteEdition } from '../../api/editions';
+import axios from 'axios';
 
 interface EditionCardProps {
   edition: Edition;
   subjectId: number;
   withBottomBorder: boolean;
 }
 
+// TODO: subject will not update after delete -> refresh page needed

```

Comment:
Instead of refreshing whole page couldn't we just refetch editions in ```subject.tsx```

Reply:
yes, we can and we should, will be corrected in next pull request - not enough time to manage state<Editions[]>

Reply:
We should, some kind of action is needed to trigger fetch, i wrote refresh, because it will cause another render. Refetch is a good idea - will be corrected later